app/views/tag/_topicCard.html.erb

```diff
@@ -32,7 +34,7 @@
       </div>
 
       <div class="card-footer" style="background-color: inherit; border:none;">
-        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= Tag.counter(tag.name)-shown_nids.count - Tag.find_nodes_by_type(tag.name, type = 'note', limit = 3).where.not(nid: shown_nids).count %> <%= translation('tag.index.more_posts') %> &raquo;</a>
+        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= Tag.counter(tag.name) - count %> <%= translation('tag.index.more_posts') %> &raquo;</a>
```

Comment:
OK, making suggestion here:

```suggestion
        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= [Tag.counter(tag.name) - shown_nids.count - Tag.find_nodes_by_type(tag.name, type = 'note', limit = 3).where.not(nid: shown_nids).count, 0].max %> <%= translation('tag.index.more_posts') %> &raquo;</a>
```

Is that right?

Reply:
![Screenshot from 2020-07-01 08-50-07](https://user-images.githubusercontent.com/41363633/86199676-5b45eb80-bb78-11ea-98e4-63bbca882716.png)

![Screenshot from 2020-07-01 08-50-48](https://user-images.githubusercontent.com/41363633/86199802-ae1fa300-bb78-11ea-97a4-821c56424c27.png)

As you can see, there are 7 research notes, 1 question and 3 wiki pages which is equal to 11 and the topic card shows 2 more posts. Also, none of the posts with tag two is shown in topic card one. So, It should have shown 8 more posts.
P.S. This is after the changes you suggested.

Reply:
What I feel is we should subtract **Tag.find_nodes_by_type(tag.name, type = 'note').where(nid: shown_nids).count** from the total count.  What it will do is count all the posts which are already shown in the topic cards above it.

Reply:
Apologies for the slow reply!

OK, let's see:

total should be 11, agreed!

so the current logic will do: `[Tag.counter(tag.name) - shown_nids.count - Tag.find_nodes_by_type(tag.name, type = 'note', limit = 3).where.not(nid: shown_nids).count, 0].max` which doesn't really make sense... i wonder. 

It used to be: `Tag.counter(tag.name)-shown_nids.count - Tag.find_nodes_by_type(tag.name, type = 'note', limit = 3).where.not(nid: shown_nids).count`

Let's try to write it in English:

(all nodes with that tag) - (count of nodes we've shown on the page anywhere) - (all nodes with that tag except for the shown nodes)

Yeah, I can't figure out why it's like that! 

You're right, it should just be:

```ruby
Tag.counter(tag.name) - Tag.find_nodes_by_type(tag.name, type = 'note').where(nid: shown_nids).count
```

Let's try that! I'm also going to rebase it so we get it to build in GitPod and we can test it out there too.